### PR TITLE
Add gulp script to auto install new theme on changes

### PR DESCRIPTION
**Pull request by krokofant**
`gulp dev` provides a watch mode with auto install

Currently the auto install only works on Windows since I don't know where themes are installed on other platforms

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,47 +1,87 @@
...
+    zip("Ursine_Cyrillic.zip")
+  );
+
+  console.log(`Building releases...`);
+  return merge(regular, cyrillic).pipe(dest("./release"));
+};
+
+const dev = () => {
+  let themeLocation;
+  switch (os.type()) {
```
Comment (aCluelessDanny) [inline gulpfile.js]: From what I can tell, themes in MacOS are stored in `~/Library/Application Support/abnerworks.Typora/themes`, where `~` is the user's home folder. Not sure how `process.env` works here, but the case for Mac would be along the lines of this:

```js
case "Darwin":
	themeLocation = "~/Library/Application Support/abnerworks.Typora/themes"
	break;
```

Not sure about Linux though

**Comment by krokofant** [comment]
@aCluelessDanny Since these are dev tools it's okay if it works on the device that the devs use. If it doesn't work on Linux that's a issue for a later time. Please merge and we can improve it from there.

File: gulpfile.js (hunk 1 above)
Comment (AdrianBylogic) [reply gulpfile.js]: Theme folder location in linux: `~/.config/Typora/themes`

Comment (AdryDev92) [reply gulpfile.js]: Theme folder location in linux: `~/.config/Typora/themes`

**Comment by AdryDev92** [comment]
> `gulp dev` provides a watch mode with auto install
> 
> Currently the auto install only works on Windows since I don't know where themes are installed on other platforms

Theme folder location in linux: `~/.config/Typora/themes`

**Comment by aCluelessDanny** [comment]
Noted. I'll put that in the next release~

**Pull request merged**